src/lib.rs

```diff
@@ -89,7 +89,12 @@ none of them fulfill these desiderata.
 #[macro_use]
 extern crate alloc;
 
-pub use crate::hash_matrix::{HashMatrix, constmatmul};
+// Re-export digest traits
+pub use digest::{
```

Comment:
I think adding `self` here will re-export `digest` itself, which is probably good to do.